src/Symfony/Component/VarDumper/Server/ContextProvider/CliContextProvider.php

```diff
@@ -9,14 +9,16 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Component\VarDumper\Dumper\ContextProvider;
+namespace Symfony\Component\VarDumper\Server\ContextProvider;
```

Comment:
should we actually rename these classes in a patch release ? they're not marked on `@internal`